lib/propolis/src/enlightenment/hyperv/hypercall.rs

```diff
@@ -39,6 +39,12 @@ impl std::fmt::Debug for MsrHypercallValue {
 }
 
 impl MsrHypercallValue {
+    /// Yields the guest page number (the PFN) at which the guest would like the
+    /// hypercall page to be placed.
+    pub fn gpfn(&self) -> u64 {
+        self.0 >> PAGE_SHIFT
+    }
```

Comment:
nit: now that the `Pfn` newtype is crate-public, perhaps this should be:
```suggestion
    /// Yields the guest page number (the PFN) at which the guest would like the
    /// hypercall page to be placed.
    pub fn gpfn(&self) -> Pfn {
        Pfn::new(self.0 >> PAGE_SHIFT)
    }
```

Reply:
Also, perhaps the `Debug` impl above that currently includes the GPA for `MsrHypercallValue` ought to also include the PFN?

Reply:
I switched to using the wrapper in more places in d378e33. I left the `Debug` impl as-is, though; anyone who wants to get a 4K PFN from a logged GPA can just put a thumb over the low three nibbles :)